lib/index.js

```diff
@@ -45,7 +43,15 @@ Drip.prototype.identify = function(identify, fn){
     .type('json')
     .send({ subscribers: [subscriber] })
     .end(function(err, res){
-      if (err) return fn(err);
+      if (err) {
```

Comment:
I've removed all that. Removed the original  "if (err) return fn(err);" and tests pass.